src/app/config/config.c

```diff
@@ -2640,7 +2646,8 @@ options_trial_assign(config_line_t *list, unsigned flags, char **msg)
   /* we liked it. put it in place. */
   rv = SETOPT_OK;
  done:
-  in_option_validation = 0;
+  log_notice(LD_GENERAL, "==== saying %d because status is %d", rv, status);
```

Comment:
Is this a helpful notice-level message?
Should we translate the numbers into descriptive strings?